marketplace-elements.styl

```diff
@@ -328,3 +328,51 @@
         width: (2 / 3 * 100)%;
     }
 }
+
+.mkt-section {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    position: relative;
+
+    &:after {
+        content: '';
+        display: block;
+        clear: both;
+    }
+}
+
+.mkt-section[full] {
```

Comment:
I'm not sure what's better but I had set this up so you can add a class instead of using attributes for this. That way you could do `.mkt-section-full { ... }` instead. I'm not sure if it's faster or if having less specificity is necessarily better but it's easy to do if you want to.

This will also add a property called `full` to the element and the class will be added and removed automatically when the attribute it added and removed. Since the markup changes that might have undesired behaviour. I'd like us to have our elements handle attributes changing but so far everything has been static so that isn't a big deal.

See https://github.com/mozilla/marketplace-elements/blob/7812f927de9fd1dcd1fcdecbd4a5c1d1fe2a9a47/marketplace-elements.js#L73.